CHANGELOG.md

```diff
@@ -3,6 +3,51 @@ All notable changes to KoalaBot will be documented in this file.
 A lot these commands will only be available to administrators
 
 ## [Unreleased]
```

Comment:
Add to the CHANGELOG all new commands and functionality (see v0.1.0 for reference)
Also make sure you update from master again, TextFilter stuff should be here